artifactory/services/download.go

```diff
@@ -133,7 +143,7 @@ func produceTasks(items []utils.ResultItem, downloadParams DownloadParams, produ
 		}
 		if v.Type != "folder" {
 			// Add a task, task is a function of type TaskFunc which later on will be executed by other go routine, the communication is done using channels.
```

Comment:
"// Add a task, task is a..." --> "// Add a task. A task is a..."

Reply:
Changed.